webApp-backend/src/main/java/org/ApeBodima/webApp_backend/config/security/SecurityConfig.java

```diff
@@ -35,7 +35,7 @@ public SecurityFilterChain securityFilterChain(HttpSecurity http) throws Excepti
         http
 
                 .authorizeHttpRequests((request) -> request
-                        .requestMatchers("/api/v*/registration/user").permitAll()
+                        .requestMatchers("/**").permitAll()
```

Comment:
you have changed the security configuration
**Please rollback it to first state**
it is essential

Reply:
ok